src/main/java/org/minions/devfund/alejandro/MainWhackAMole.java

```diff
@@ -0,0 +1,54 @@
+package org.minions.devfund.alejandro;
+import java.util.Scanner;
+
+/**
+ * class for main wack a mole game.
+ */
+public interface MainWhackAMole {
+
+    /**
+     * This is the main method to start running the game.
+     * @param args arguments.
+     */
+    static void main(final String[] args) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "args". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)